atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/transaction/knowledge/DefaultKnownAbortedTransactionsTest.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * (c) Copyright 2022 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.transaction.knowledge;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.*;
+
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Range;
+import com.palantir.atlasdb.AtlasDbConstants;
+import com.palantir.atlasdb.transaction.knowledge.AbortedTransactionSoftCache.TransactionSoftCacheStatus;
+import com.palantir.tritium.metrics.registry.DefaultTaggedMetricRegistry;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.LongStream;
+import org.junit.Test;
+
+public final class DefaultKnownAbortedTransactionsTest {
+    private final FutileTimestampStore futileTimestampStore = mock(FutileTimestampStore.class);
+    private final AbortedTransactionSoftCache softCache = mock(AbortedTransactionSoftCache.class);
+    private final DefaultKnownAbortedTransactions knownAbortedTransactions =
+            new DefaultKnownAbortedTransactions(futileTimestampStore, softCache, new DefaultTaggedMetricRegistry());
+
+    @Test
+    public void testIsKnownAbortedReturnsTrueIfAbortedInSoftCache() {
+        when(softCache.getSoftCacheTransactionStatus(anyLong())).thenReturn(TransactionSoftCacheStatus.IS_ABORTED);
+        long abortedTimestamp = 27L;
```

Comment:
- Probably refactor this into a constant.
- `when(...)` should reference this number, especially since we don't validate the calls later.